runners_manager/monitoring/prometheus.py

```diff
@@ -0,0 +1,11 @@
+"""Global class to handle all Prometheus metrics."""
+
+from prometheus_client import Enum
+
+
+class Metrics(object):
+    runner_status = Enum(
+        'openstack_actions_runner_status', 'Metrics displaying the status of a runner',
+        states=['creating', 'deleting', 'respawning', 'online', 'running', 'offline'],
+        labelnames=['name', 'flavor', 'image'],
+    )
```

Comment:
This class should evolve with function when it growth

Reply:
Indeed, I agree, especially when reusing labels and all. Once we add more metrics we should keep that in mind.